### PR TITLE
CP-1

### PR DESCRIPTION
**Pull request by githubTestUser20**
CP-1 pul request description

### CODE REVIEW

**Review by githubTestUser20 — COMMENTED** [review]
Starting code review

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # PR_TEST
 test
+
+CP-1
```
Comment (githubTestUser20) [inline README.md]: you should update the following line

Comment (githubTestUser20) [reply README.md]: ok

**Review by githubTestUser20 — COMMENTED** [review]
Ok perfect

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # PR_TEST
 test
+
+CP-1 (updated line as requested)
```
Comment (githubTestUser20) [inline README.md:4]: I'm still not sure about it, add more lines

**Review by githubTestUser20 — COMMENTED** [review]
Ok seems fine.

File: README.md (hunk 2 above)
Comment (githubTestUser20) [reply README.md:4]: test

Comment (githubTestUser20) [reply README.md:4]: dsa

Comment (githubTestUser21) [reply README.md:4]: We should add more lines

File: newFile.md [hunk 3]
```diff
@@ -0,0 +1 @@
+### new File MD
```
Comment (githubTestUser21) [inline newFile.md:1]: Please add more content

**Review by githubTestUser21 — CHANGES_REQUESTED** [review]

**Review by githubTestUser21 — APPROVED** [review]

**Pull request merged**